docs/workers.md

```diff
@@ -178,8 +178,10 @@ recommend the use of `systemd` where available: for information on setting up
 
 ### `synapse.app.generic_worker`
 
-This worker can handle API requests matching the following regular
-expressions:
+This worker can handle API requests matching the following regular expressions.
+These endpoints can be routed to any worker. If a worker is set up to handle a
+stream then additional endpoints may be required to route to said worker, refer
+to the [stream writers](#stream-writers) section below for further information.
```

Comment:
I'm a bit confused about this. Are we saying that `sendToDevice` and things can go to any `generic_worker`, *unless* there is a worker which is defined to handle the `to_device` stream, in which case you have to route `sendToDevice` to that specific worker?